runtime/moonbase/src/weights/pallet_utility.rs

```diff
@@ -106,4 +106,14 @@ impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 			.saturating_add(Weight::from_parts(5_619_091, 0).saturating_mul(c.into()))
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 	}
+
+    fn dispatch_as_fallible() -> Weight {
+        // TODO: regenerate weights
+        <()>::dispatch_as_fallible()
```

Comment:
Lets try to generate this locally, just to be sure the benchmarks are not broken